.github/workflows/awstest.yml

```
@@ -1,31 +1,30 @@
-name: nf-core aws test
+name: AWS Megatests
 # This workflow is triggered on PRs to the master branch.
 # It runs the -profile 'test' on AWS batch
+
 on:
   pull_request:
     branches:
-      - master
+    - 'master'
   release:
     types: [published]
 
 jobs:
-    Awstest:
-        runs-on: ${{ matrix.os }}
-        strategy:
-            matrix:
-                os: ['ubuntu-latest']
-                python-version: ['3.7']
-        steps:
-        - uses: goanpeca/setup-miniconda@v1
-          with:
-            auto-update-conda: true
-            python-version: ${{ matrix.python-version }}
-        - name: Install aws client
-          run: conda install -c conda-forge awscli
-        - name: Start AWS batch job
-          env:
-            AWS_ACCESS_KEY_ID: ${{secrets.AWS_KEY_ID}}
-            AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_KEY_SECRET}}
-            TOWER_ACCESS_TOKEN: ${{secrets.TOWER_ACCESS_TOKEN}}
-          run: |
-            aws batch submit-job --region eu-west-1 --job-name nf-core-sarek --job-queue 'default-8b3836e0-5eda-11ea-96e5-0a2c3f6a2a32' --job-definition nextflow --container-overrides command=nf-core/sarek,"-r ${GITHUB_SHA} -profile test --outdir s3://nf-core-awsmegatests/sarek/results-${GITHUB_SHA} -w s3://nf-core-awsmegatests/sarek/work-${GITHUB_SHA}"
+  run-awstest:
+    name: Run AWS test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Miniconda
+        uses: goanpeca/setup-miniconda@v1.0.2
+        with:
+          auto-update-conda: true
+          python-version: 3.7
+      - name: Install awscli
+        run: conda install -c conda-forge awscli
+      - name: Start AWS batch job
+        env:
+          AWS_ACCESS_KEY_ID: ${{secrets.AWS_KEY_ID}}
+          AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_KEY_SECRET}}
+          TOWER_ACCESS_TOKEN: ${{secrets.TOWER_ACCESS_TOKEN}}
+        run: |
+          aws batch submit-job --region eu-west-1 --job-name nf-core-sarek --job-queue 'default-8b3836e0-5eda-11ea-96e5-0a2c3f6a2a32' --job-definition nextflow --container-overrides command=nf-core/sarek,"-r ${GITHUB_SHA} -profile test --outdir s3://nf-core-awsmegatests/sarek/results-${GITHUB_SHA} -w s3://nf-core-awsmegatests/sarek/work-${GITHUB_SHA}" 
```

That will add tower as well

```suggestion
          aws batch submit-job --region eu-west-1 --job-name nf-core-sarek --job-queue 'default-8b3836e0-5eda-11ea-96e5-0a2c3f6a2a32' --job-definition nextflow --container-overrides '{"command": ["nf-core/sarek", "-r ${GITHUB_SHA} -profile test --outdir s3://nf-core-awsmegatests/sarek/results-${GITHUB_SHA} -w s3://nf-core-awsmegatests/sarek/work-${GITHUB_SHA} -with-tower"], "environment": [{"name": "TOWER_ACCESS_TOKEN", "value": "'"$TOWER_ACCESS_TOKEN"'"}]}'
```